e2e-tests/.ci/server.prepare.sh

```diff
@@ -63,7 +61,7 @@ for SERVICE in $ENABLED_DOCKER_SERVICES; do
     fi
     mme2e_log "Configuring the $SERVICE container"
     ${MME2E_DC_SERVER} exec -T openldap bash -c 'ldapadd -Y EXTERNAL -H ldapi:/// -w mostest || true' <../../server/tests/custom-schema-objectID.ldif
-    ${MME2E_DC_SERVER} exec -T -- openldap bash -c 'ldapadd -Y EXTERNAL -H ldapi:/// -w mostest || true' < ../../server/tests/custom-schema-cpa.ldif
+    ${MME2E_DC_SERVER} exec -T -- openldap bash -c 'ldapadd -Y EXTERNAL -H ldapi:/// -w mostest || true' <../../server/tests/custom-schema-cpa.ldif
```

Comment:
we've removed some spaces (here and elsewhere) -- is that ok? probably should be left, if only for readability?